misc/webscraper.py

```diff
@@ -31,7 +31,7 @@ async def get_links(url: str):
         )
         await browser.close()
 
-    return links
+    return set(links)
```

Comment:
We're hitting rate limits from Slack posting so many links. This will reduce the many duplicates we attempt to post, and hopefully alleviate the rate-limiting issues.